demisto_sdk/commands/format/tests/test_formatting_json_test.py

```diff
@@ -503,6 +504,42 @@ def test_update_id(self, layoutscontainer_formatter):
         layoutscontainer_formatter.update_id()
         assert layoutscontainer_formatter.data['name'] == layoutscontainer_formatter.data['id']
 
+    @pytest.mark.parametrize('schema', [GENERICFIELD_SCHEMA_PATH,
+                                        INCIDENTFIELD_SCHEMA_PATH,
+                                        INDICATORFIELD_SCHEMA_PATH])
+    def test_remove_null_doesnt_remove_defaultrows(self, schema):
+        """
+        Given
+            - Generic, indicator and incident fields schemes
+        When
+            - Run format on Generic, indicator and incident fields files
```

Comment:
This is not accurate, you are not running the format command but the "remove_null_fields" method